tests/phpunit/Logger/GitHubAnnotationsLoggerTest.php

```diff
@@ -71,4 +87,16 @@ public static function metricsProvider(): iterable
             ],
         ];
     }
+
+    public function test_it_logs_correctly_with_ci_github_workspace(): void
+    {
+        \Safe\putenv('GITHUB_WORKSPACE=/my/project/dir');
+        self::$pathPrefix = '/my/project/dir/';
```

Comment:
What do you think about resetting the path prefix during the tear down?

I couldn't find if it breaks anything now, but I can imagine it could get into way when adding another test.

And please don't force-push things: we are going to squash-merge everything anyway.

Reply:
Since tests are executed in a random order, shouldn't it already break then?

I can do the change later today, not behind a computer right now.